src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+$days-count: 31;
+$day-block-size: 100px;
+$days-name: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
```

Comment:
let's split your main file into separate files:

in the `styles` folder, create `utils ` folder and put 2 files there:
variables.scss and reset.scss with `body { margin: 0; }`